src/solver/variable/include/antares/solver/variable/container.hxx

```diff
@@ -254,7 +251,7 @@ void List<NextT>::buildSurveyReport(SurveyResults& results,
 
     // The new filename
     results.data.filename.clear();
-    results.data.filename << results.data.output << SEP;
+    results.data.filename << std::filesystem::path(std::string(results.data.output)) / "";
```

Comment:
This is ugly...

Reply:
I agree, but it avoids intermediate variables needed to convert from yuni string